sdk/identity/azure-identity/src/main/java/com/azure/identity/AzureIdentityEnvVars.java

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity;
+
+import com.azure.core.util.ExpandableStringEnum;
+
+import java.util.Collection;
+
+/**
+ * <p>Defines well-known Azure Identity environment variable names that can be used with 
+ * {@link DefaultAzureCredentialBuilder#requireEnvVars(AzureIdentityEnvVars...)}.</p>
+ *
+ * <p>This expandable enum provides a type-safe way to reference common Azure Identity environment
+ * variables while still allowing for custom environment variable names.</p>
+ *
+ * @see DefaultAzureCredentialBuilder#requireEnvVars(AzureIdentityEnvVars...)
+ */
+public final class AzureIdentityEnvVars extends ExpandableStringEnum<AzureIdentityEnvVars> {
+
+    public AzureIdentityEnvVars() {
```

Comment:
Mark this ctor as deprecated. This is the pattern we are following across all ExpandableStringEnum types.